Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM jupyter/datascience-notebook
```

Comment:
This file is not strictly part of the extension, but we can leave it here for now in case we need to deploy the extension quickly. If we integrate this extension in the mono repo, it should be removed though.